easy_my_coop_loan_website/controllers/main.py

```diff
@@ -0,0 +1,94 @@
+from odoo import http
+from odoo.http import request
+
+from odoo.tools.translate import _
+
+
+class WebsiteLoanIssueSubscription(http.Controller):
+
+    @http.route(['/subscription/get_loan_issue'],

```

Comment:
`subscription/loan_issues` would be a closer naming to classic api paths.

Tu peux pas faire un get pour un call ajax?

Reply:
except that this route return one loan_issue based on its id.

this is a json call so what do you mean by ajax ?

Reply:
You use this route in an ajax call (js calling the backend). I have a vague souvenir that you must use post methods in those calls. But if not, a get with url parameter would be more appropriate. `GET` to get data, `POST` to post data.

Reply:
Ok I changed it to GET

Reply:
Great, the js is still working?

Reply:
no